[Quant][PT2E] Remove x86 inductor pt2e backend config

leslie-fang-intel opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #105456
* #105455
* #104590
* #104588
* #104581
* #104580
* #104573
* __->__ #105039
* #105109

**Summary**
For the Quantization PT2E path, we recommend to use `X86InductorQuantizer` instead of backend config of `x86_inductor_pt2e_backend_config`. Remove the `x86_inductor_pt2e_backend_config` and the relevant testing.

leslie-fang-intel committed:
Remove test of _x86_inductor_pt2e backend config

[ghstack-poisoned]

leslie-fang-intel commented:
Hi @jerryzh168, could you help to take look of this PR?

leslie-fang-intel committed:
Update on "[Quant][PT2E] Remove x86 inductor pt2e backend config"


**Summary**
For the Quantization PT2E path, we recommend to use `X86InductorQuantizer` instead of backend config of `x86_inductor_pt2e_backend_config`. Remove the `x86_inductor_pt2e_backend_config` and the relevant testing.


[ghstack-poisoned]

jerryzh168 commented:
looks good, please wait a bit, I have a long stack of refactoring PRs that I want to land, since they changed both internal and external files I need to wait until there are no changes to quant related code base

leslie-fang-intel committed:
Update on "[Quant][PT2E] Remove x86 inductor pt2e backend config"


**Summary**
For the Quantization PT2E path, we recommend to use `X86InductorQuantizer` instead of backend config of `x86_inductor_pt2e_backend_config`. Remove the `x86_inductor_pt2e_backend_config` and the relevant testing.


[ghstack-poisoned]

leslie-fang-intel committed:
Update on "[Quant][PT2E] Remove x86 inductor pt2e backend config"


**Summary**
For the Quantization PT2E path, we recommend to use `X86InductorQuantizer` instead of backend config of `x86_inductor_pt2e_backend_config`. Remove the `x86_inductor_pt2e_backend_config` and the relevant testing.


[ghstack-poisoned]

leslie-fang-intel committed:
Update on "[Quant][PT2E] Remove x86 inductor pt2e backend config"


**Summary**
For the Quantization PT2E path, we recommend to use `X86InductorQuantizer` instead of backend config of `x86_inductor_pt2e_backend_config`. Remove the `x86_inductor_pt2e_backend_config` and the relevant testing.


[ghstack-poisoned]

leslie-fang-intel commented:
@pytorchbot merge
